modules/cpr/src/main/java/org/atmosphere/cpr/BroadcasterCache.java

```diff
@@ -84,6 +84,18 @@
      * and some future must be cancelled. This method will always be invoked when a {@link Broadcaster} gets destroyed.
      */
     void cleanup();
+    
+    /**
+     * Get serializer for cache messages.
+     * @return The {@link Serializer}.
+     */
+    Serializer getCacheSerializer();
```

Comment:
I was unable to properly fix it myself, any help would be highly appreciated. :)